### PR TITLE
Feat/transaction estimate2

### PR DESCRIPTION
**Pull request by ieow**
Add Transaction Estimation for send transaction

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
No estimation

Issue Number: N/A
## What is the new behavior?

Will show estimation account post transaction 

-
-
-

### CODE REVIEW

File: src/pages/Confirm.vue [hunk 1]
```diff
@@ -67,27 +105,65 @@ onMounted(async () => {
       redirectToResult(jsonrpc, { message: "Invalid or Missing Params", method }, req_id, resolveRoute);
       return;
     }
+
     origin.value = txData.origin as string;
     network.value = txData.network || "";
+    const networkConfig = txData.networkDetails as ProviderConfig;
```
Comment (ArpitAgrawal321) [inline src/pages/Confirm.vue:111]: `txData.networkDetails` is not defined for redirect flow, use wallets providerConfig if txData doesn't have it.

File: src/pages/Confirm.vue [hunk 2]
```diff
@@ -171,27 +248,57 @@ const rejectTxn = async () => {
...
+      :decoded-inst="decodedInst || []"
+      :origin="origin"
+      :estimation-in-progress="estimationInProgress"
+      :estimated-balance-change="estimatedBalanceChange"
+      :has-estimation-error="hasEstimationError"
+      :sign-tx-only="signTxOnly"
+      :tx-fee="finalTxData.totalNetworkFee"
+      :is-gasless="finalTxData.isGasless"
+      :network="network"
+      @on-close-modal="rejectTxn()"
```
Comment (ArpitAgrawal321) [inline src/pages/Confirm.vue:284]:
```suggestion
      @on-close-modal="closeModal()"
```

File: src/pages/ConfirmNft.vue [hunk 3]
```diff
@@ -51,6 +87,9 @@ async function cancelTransfer() {
     :receiver-pub-key="params.receiver_add"
     :sender-pub-key="params.sender_add"
     :token="selectedNft"
+    :estimation-in-progress="estimationInProgress"
+    :estimated-balance-change="estimatedBalanceChange"
+    :has-estimation-error="hasEstimationError"
     @transfer-confirm="confirmTransfer"
     @transfer-cancel="cancelTransfer"
```
Comment (ArpitAgrawal321) [inline src/pages/ConfirmNft.vue:94]:
```suggestion
    @transfer-reject="cancelTransfer"
```

**Comment by ieow** [comment]
Breakdown to smaller PR implementation #309 #310

**Pull request closed without merging**